services/debugger/uploader.go

```diff
@@ -126,13 +130,12 @@ func (uploader *Uploader) uploadEvents(eventBuffer []interface{}) {
 			// Refresh the connection
 			continue
 		}
-		defer func() { httputil.CloseResponse(resp) }()
+		resp.Body.Close()
```

Comment:
both read and close response body always
```suggestion
		httputil.CloseResponse(resp)
```

Reply:
Doing this didn't pass the linter and so had to modify it the current way

Reply:
the following should do the trick and I will eventually treat this issue with the new linter
```
func(){httputil.CloseResponse(resp)}()
```